pkg/api/validation/validation.go

```diff
@@ -30,12 +30,32 @@ import (
 	"github.com/golang/glog"
 )
 
+const minPort int = 1
+const minHostPort int = 0
+const maxPort int = 65536
+
+const QualifiedNameErrorMsg string = "must match regex [" + util.Dns1123SubdomainFmt + " / ] " + util.Dns1123LabelFmt
+const PortNameErrorMsg string = "must be less than %d characters and match regex " + util.Dns1123LabelFmt
```

Comment:
Oh, I missed this %d. You're right, as long as I can substitute check in appropriate if stmt with util.Dns1123LabelLength constant.